src/BizHawk.Client.EmuHawk/CustomControls/HexTextBox.cs

```diff
@@ -135,6 +135,21 @@ protected override void OnTextChanged(EventArgs e)
 			base.OnTextChanged(e);
 		}
 
+		protected override void OnPaste(PasteEventArgs e)
+		{
+			if (e.ContainsText)
+			{
+				string text = e.Text.Trim().RemovePrefix("0x").RemovePrefix('$');

```

Comment:
This could be deduped and perhaps turned into a more robust utility method. Wasn't sure if it was worth it and where to put it, if so.